examples/Quick_Starts/Automated_Meshing/om6Case.json

```diff
@@ -0,0 +1,54 @@
+{
+    "geometry" : {
+        "refArea" : 1.15315084119231,
+        "momentCenter" : [0.0, 0.0, 0.0],
+        "momentLength" : [1.47602, 0.801672958512342, 1.47602]
+    },
+    "volumeOutput" : {
+        "outputFormat" : "tecplot",
+        "animationFrequency" : -1,
+        "outputFields": ["primitiveVars", "Mach"]
+    },
+    "surfaceOutput" : {
+        "outputFormat" : "tecplot",
+        "animationFrequency" : -1,
+        "outputFields": ["primitiveVars", "Cp", "Cf"]
+    },
+    "navierStokesSolver" : {
+        "absoluteTolerance" : 1e-10,
+        "kappaMUSCL" : -1.0
+    },
+    "turbulenceModelSolver" : {
+        "modelType" : "SpalartAllmaras",
+        "absoluteTolerance" : 1e-8
+    },
+    "freestream" :
+    {
+        "Reynolds" : 14.6e+6,
```

Comment:
After Maciej is done with the unit stuff, perhaps we  can show the user how to calculate `Reynolds` using the new Python API?